server/app/services/JiraWriterService.java

```diff
@@ -0,0 +1,159 @@
+package services;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import models.ResponseToClient;
+import play.libs.Json;
+
+/**
+ * This class for any operation which makes changes on JIRA, that is, "writing operations".
+ * Currently not implemented at all.
```

Comment:
Please add `TODO: ` tag for this class, since they are not implemented yet, but it will be in the future.